src/libLookup/Lookup.cpp

```diff
@@ -1592,15 +1583,6 @@ bool Lookup::ProcessSetDSBlockFromSeed(const bytes& message,
         continue;
       }
       m_mediator.m_dsBlockChain.AddBlock(dsblock);
-      // Store DS Block to disk
-      if (!ARCHIVAL_NODE) {
```

Comment:
the block should be stored if it is not archival node

Reply:
added it back